### ISSUE TITLE
The repl recording in README.md is archived

### ISSUE THREAD

**Issue by rbtcollins** [box-builder/box#270]
https://asciinema.org/a/c1n0h0g73f10x4cuzjf1i51vg reports that it is archived.

**Comment by erikh** [comment]
Yeah I haven’t had too much time to pay attention to this lately. I still
use the project in places but it’s a but stale from a development
perspective.
